### ISSUE TITLE
Widget Corner Radius

### ISSUE THREAD

**Issue by phikal** [WangDaYeeeeee/GeometricWeather#416]
Would it be possible to configure the radius of the widgets?  Either globally or each time a widget is placed?

**Comment by papjul** [comment]
Can you show some examples of this configuration from other apps? So we can have a better idea on how it could be implemented.

**Comment by phikal** [comment]
No, sadly I don't have any other examples.  Maybe this could also be simplifed by just providing either the current radius or square widgets?  The reasion I am bringing this up is because I have exactly two widgests on my home screen and one is perfectly square while the other one (Geometric Weather) isn't, and I think it would look better if it were.
